books/Frankenstein.py

```diff
@@ -1 +1,2 @@
-print("hello")
\ No newline at end of file
+print("hello")
```

Comment:
Please review this

Reply:
[2f81904](https://github.com/mesohn/bookbot1/pull/1/commits/2f819043dcb673c3411f2f9241ca3b86b541c3d3)